io.openliberty.java.internal_fat_misc/java17/settings.gradle

```diff
@@ -0,0 +1,14 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+// To create this war file (ex: ./gradlew build) for newer versions of Java (typically early access ones) before gradle supports it
+// See the README.md file
+
+rootProject.name = 'java17-app'
```

Comment:
Should be moved to a root settings file.

Reply:
Done, thanks for the help!